### ISSUE TITLE
`saveDef`, `lookupDef` replacements?

### ISSUE THREAD

**Issue by conal** [ku-fpg/hermit#152]
Am I right in thinking that `saveDef` and `lookupDef` became `rememberR` and `unfoldRememberedR`?
With `unfoldRememberedR`. If so, for this purpose (saving & recalling), should I pass `NotUsed` to `unfoldRememberedR`?

**Comment by xich** [comment]
Yes (to everything you said)... though looking at it now, I'm not sure why I even require that argument, since all remembered definitions are always assumed true, there is no real point in tracking their used status. I'll think about that some and probably clean up the remember-ing API (again, sorry!) at some point. But I'll let you know when I do.
